components/style/properties/longhand/font.mako.rs

```diff
@@ -350,6 +350,58 @@ ${helpers.single_keyword("font-variant",
     }
 </%helpers:longhand>
 
+// https://www.w3.org/TR/css-fonts-3/#font-size-adjust-prop
+// FIXME: This prop should be animatable
+<%helpers:longhand products="none" name="font-size-adjust" animatable="False">
+    use values::CSSFloat;
+    use values::NoViewportPercentage;
+    use values::computed::ComputedValueAsSpecified;
+
+    impl ComputedValueAsSpecified for SpecifiedValue {}
+    impl NoViewportPercentage for SpecifiedValue {}
+
+    #[derive(Clone, Debug, PartialEq)]
+    #[cfg_attr(feature = "servo", derive(HeapSizeOf))]
+    pub enum SpecifiedValue {
+        None,
+        Number(CSSFloat),
+    }
+
+    pub mod computed_value {
+        use cssparser::ToCss;
```

Comment:
As of #14089, we make use of `style_traits::ToCss`. This was a recent change, and the guide should be updated :)